bin/pycbc_live

```diff
@@ -537,13 +543,18 @@ class LiveEventManager(object):
         logging.info('Coincident candidate! Saving as %s', fname)
 
         # Verbally explain some details not obvious from the other info
-        comment = ('Trigger produced as a {} coincidence. '
-                   'FAR is based on all listed detectors.<br />'
-                   'Two-detector ranking statistic: {}<br />'
-                   'Followup detectors: {}')
-        comment = comment.format(ppdets(coinc_ifos),
-                                 args.ranking_statistic,
-                                 ppdets(followup_ifos))
+        comment = (
+            'Trigger produced as a {} coincidence.<br />'
+            'Two-detector ranking statistic: {}<br />'
+            'Detectors used for FAR calculation: {}.<br />'
+            'Detectors used for localization: {}.'
+        )
+        comment = comment.format(
+            ppdets(coinc_ifos),
+            args.ranking_statistic,
+            set(ifos) - self.skymap_only_ifos,
+            ppdets(followup_ifos)
+        )
```

Comment:
```followup_ifos``` contribute to FAR but probably are not ```coinc_ifos```. @titodalcanton can confirm. Also cannot ```self.skymap_only_ifos``` be put for ```'Detectors used for localization: {}.'```

Reply:
The final set should be self.skymap_only_ifos.

If we imagine that we are using HL for coincidences, V for followup and K for skymap-only, then it would currently print Virgo twice.

If being pedantic, the coinc IFOs are also used for localisation, so it should read 'Detectors used for localization only' (or similar)

Reply:
Imagine we are using a network of detectors ABCD. A and B have the same sensitivity. C is slightly less sensitive. D is significantly less sensitive, and configured as a skymap-only detector. In this case, the most likely candidate will result from a coincidence between A and B, however the FAR will be calculated using A, B and C (using the usual method from Dal Canton et al 2021), and the localization will be done using A, B, C and D.

The variable name `followup_ifos` is a bit confusing here, because it basically lists all detectors that will be used for localization, but *a subset of those* might also be used for improving the FAR.

I think the GraceDB comment I am suggesting is appropriate though. The last line is meant to report *all* detectors used for localization, which I think is more useful than reporting those *only* used for localization. But I can add that as another line if useful.

Reply:
I have added another line that lists the detectors configured for localization only.